src/index.js

```diff
@@ -1,54 +1,16 @@
-import React from 'react';
```

Comment:
Changes to this file are limited to moving the core initialization out to `bootstrap.js`.